extensions/sql-migration/src/dialog/tdeConfiguration/tdeConfigurationDialog.ts

```diff
@@ -130,6 +136,24 @@ export class TdeConfigurationDialog {
 			adsMethodButton.onDidChangeCheckedState(async checked => {
 				if (checked) {
 					this.migrationStateModel.tdeMigrationConfig.setPendingTdeMigrationMethod(ConfigDialogSetting.ExportCertificates);
+					let validationTitleData = await this.migrationStateModel.getTdeValidationTitles();
+
+					let networkPathValidated =
+						(this.migrationStateModel.tdeMigrationConfig.getPendingNetworkPath() !== '') &&
```

Comment:
Question: don’t we have any thing like string.empty or whitespace to check in typescript?

Reply:
I found an empty string constant in utils.ts.  I will use that instead.

Reply:
Actually, I'm not sure who maintains this utils file, I think it is better not to depend on it.  For now I'll keep the '' string comparison.

Reply:
Importing that caused extension to fail to run